### ISSUE TITLE
tokenInputIds feature should have configurable start and pad token ids

### ISSUE THREAD

**Issue by jobergum** [vespa-engine/vespa#28393]
TL;DR;
The START/PAD token ids for the `tokenInputIds` rank feature should be configurable. 

The [tokenInputIds](https://docs.vespa.ai/en/reference/rank-features.html#:~:text=tokenInputIds(length%2C%20input_1%2C%20input_2%2C%20...)) feature which builds the input sequence for a transformer model uses hardcoded start of sequence (101) and padding (102) which is fine for vanilla BERT models using wordpiece, but is incorrect for models using 1 and 2 as start and pad, like sentence-piece models (Deberta, etc).

**Comment by jobergum** [comment]
```
tokenizer("bmw 325i radio oem not navigation system", "AWESAFE Car <hi>Radio</hi> Stereo Android 10 for BMW 3 Series 328i 330i 335i 325i E90/E91/E92/E93 7 Inch Touch Screen Head Unit with Bluetooth Support Apple Carplay Andriod Auto")

{'input_ids': [1, 49801, 21943, 667, 2442, 83321, 298, 7322, 492, 2, 336, 21454, 79896, 3166, 2569, 8080, 1504, 33267, 4052, 320, 8080, 1504, 25603, 2893, 466, 270, 8327, 404, 3046, 36274, 667, 18061, 667, 32775, 667, 21943, 667, 829, 4108, 320, 1276, 9527, 320, 1276, 9579, 320, 1276, 9968, 574, 19891, 9622, 9114, 3796, 6184, 275, 8992, 4177, 2013, 3166, 7829, 414, 16911, 407, 3892, 2], 'token_type_ids': [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1], 'attention_mask': [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1]}
```

**Comment by lesters** [comment]
PR: https://github.com/vespa-engine/vespa/pull/28477

**Comment by lesters** [comment]
Above PR is merged.
